### PR TITLE
Pbi 257442 mqtt doc qa

### PR DESCRIPTION
**Pull request by osisoft-mbishop**

### CODE REVIEW

File: content/configuration/data-selection-generic.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ The following are examples of valid MQTT data selection configurations<sup>1</su
 ]
 ```
 
-<sup>1</sup> **Note:** Both **ValueField** and **IndexField** require the correct structure of the JSON payload to be specified, in other words, what the data source returns. The previous examples use the following Json payload structure:
+<sup>1</sup> **Note:** Both **ValueField** and **IndexField** require the correct structure of the JSON payload to be specified (in other words, what the data source returns). The previous examples use the following JSON payload structure:
```
Comment (osisoft-mbishop) [inline content/configuration/data-selection-generic.md:135]: Fixed a comma splice and capltalized "JSON".

File: content/configuration/discovery/data-source-discovery-generic.md [hunk 2]
```diff
@@ -14,12 +14,12 @@ The string of the **query** parameter must contain string items in the following
...
 |------------------|----------|-------------|
-| **Topics**       | Optional |  The topics that the adapter will subscribe to when the discovery is posted.<br>**Note:** If you want to specify multiple topics in the query, you must separate the topic names with a comma. If you do not specify a topic, the adapter will subscribe to all topics. |
-| **WaitTime**     | Optional |  The time window in which the adapter will perform the discovery by listening for specified topics. Once the wait time has elapsed, the discovery results will be available.<br><br>Minimum value: `0.00:00:30` (30 seconds)<br>Maximum value: `7.00:00:00` (7 days)<br>**Note:** If you do not specify a wait time, the default value `0.00:01:00` (1 minute) will be applied.<br>When you specify **WaitTime** with a numeric value, for example `WaitTime=45`, the adapter interprets the value as seconds instead of time span.          |
+| **Topics**       | Optional |  The topics that the adapter subscribes to when the discovery posts.<br>**Note:** If you want to specify multiple topics in the query, you must separate the topic names with a comma. If you do not specify a topic, the adapter subscribes to all topics. |
```
Comment (osisoft-mbishop) [inline content/configuration/discovery/data-source-discovery-generic.md:17]: Rewrote parts of this doc to remove past/future tense. As a group, we have issues keeping our writing in the present tense, so try to be mindful of this in the future. I'm plenty guilty of this myself.

File: content/configuration/discovery/data-source-discovery-generic.md [hunk 3]
```diff
@@ -14,12 +14,12 @@ The string of the **query** parameter must contain string items in the following
...
+| **WaitTime**     | Optional |  The time window in which the adapter performs the discovery by listening for specified topics. Once the wait time elapses, the discovery results are available.<br><br>Minimum value: `0.00:00:30` (30 seconds)<br>Maximum value: `7.00:00:00` (7 days)<br>**Note:** If you do not specify a wait time, the default value `0.00:01:00` (1 minute) is applied.<br>When you specify **WaitTime** with a numeric value, for example `WaitTime=45`, the adapter interprets the value as seconds instead of time span.          |
 
 ### Query rules
 
-The following rules apply for specifying the query string:
+The following rules apply when specifying the query string:
```
Comment (osisoft-mbishop) [inline content/configuration/discovery/data-source-discovery-generic.md:22]: This sentence was awkward.

File: content/configuration/discovery/data-source-discovery-sparkplug-b.md [hunk 4]
```diff
@@ -8,14 +8,12 @@ A discovery against the data source of an MQTT Sparkplug B adapter allows you to
...
+The string of the **query** parameter must contain string items in the following form: <br>`Topics=Topic1,Topic2;WaitTime=<WaitTime>` where the Sparkplug B topic structure is of the form `spBv1.0/<Group_Id>/<Message_Type>/<Edge_Node_Id>/[<Device_Id>]`.
 
 | String item      | Required | Description |
 |------------------|----------|-------------|
 | **Group_Id**     | Optional | This element of the Topic namespace logically groups MQTT EoN nodes into the MQTT server and the consuming MQTT clients.|
-| **Message_Type**     | Optional | This element of the Topic namespace indicates how to handle the MQTT
-payload of the message. The message type is typically `NDATA` or `DDATA`. For more information, see [Data selection (Sparkplug B)](xref:PIAdapterForMQTTSparkplugBDataSelectionConfiguration). |
+| **Message_Type**     | Optional | This element of the Topic namespace indicates how to handle the MQTT payload of the message. The message type is typically `NDATA` or `DDATA`. For more information, see [Data selection (Sparkplug B)](xref:PIAdapterForMQTTSparkplugBDataSelectionConfiguration). |
```
Comment (osisoft-mbishop) [inline content/configuration/discovery/data-source-discovery-sparkplug-b.md:16]: Table had some formatting issues.

File: content/index.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ uid: index
 
 # Overview
 
-PI Adapter for MQTT is a data-collection component that transfers time-series data from source devices to OMF endpoints in OSIsoft Cloud Services or PI Servers. MQTT (Message Queuing Telemetry Transport) is a messaging protocol created for Machine-to-Machine (M2M)/Internet of Things (IOT) communication. The adapter can connect to any device that uses the MQTT protocol for communication with constrained devices and server applications for data exchange.
+PI Adapter for MQTT is a data-collection component that transfers time-series data from source devices to OMF endpoints in OSIsoft Cloud Services, PI Servers, or Edge Data Store. MQTT (Message Queuing Telemetry Transport) is a messaging protocol created for Machine-to-Machine (M2M)/Internet of Things (IOT) communication. The adapter can connect to any device that uses the MQTT protocol for communication with constrained devices and server applications for data exchange.
```
Comment (osisoft-mbishop) [inline content/index.md:7]: Added EDS.

**Review by osisoft-mbishop — COMMENTED** [review]
Doc is in good shape. I found a few minor issues, see comments below.

**Review by TinaUll — APPROVED** [review]

**Pull request merged**